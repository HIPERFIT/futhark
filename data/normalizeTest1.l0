fun int tupfun( (int*(real*int)) x, (int*(real*int)) y ) = 
    let (x0, (x1,x2)) = x in
    let (y0, (y1,y2)) = y in
    33

fun int main() = 
    let x = ( read(int), ( read(real), read(int) ) ) in
    let y = ( read(int), ( read(real), read(int) ) ) in
    write(tupfun(x,y))
