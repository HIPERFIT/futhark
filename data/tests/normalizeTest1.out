Result of evaluation: 5
